GVFS/GVFS.Common/Maintenance/LooseObjectsStep.cs

```diff
@@ -65,15 +113,18 @@ protected override void PerformMaintenance()
                         }
                     }
 
-                    int beforeCount = this.CountLooseObjects();
+                    int beforeLooseObjectsCount = this.CountLooseObjects();
                     GitProcess.Result result = this.RunGitCommand((process) => process.PrunePacked(this.Context.Enlistment.GitObjectsRoot));
-                    int afterCount = this.CountLooseObjects();
+                    int afterLooseObjectsCount = this.CountLooseObjects();
+
+                    this.CreateLooseObjectsPackFile();
 
                     EventMetadata metadata = new EventMetadata();
                     metadata.Add("GitObjectsRoot", this.Context.Enlistment.GitObjectsRoot);
-                    metadata.Add("StartingCount", beforeCount);
-                    metadata.Add("EndingCount", afterCount);
-                    metadata.Add("RemovedCount", beforeCount - afterCount);
+                    metadata.Add("StartingCount", beforeLooseObjectsCount);
+                    metadata.Add("EndingCount", afterLooseObjectsCount);
+                    metadata.Add("RemovedCount", beforeLooseObjectsCount - afterLooseObjectsCount);
+                    metadata.Add("LooseObjectsPutIntoPackFile", this.LooseObjectsPutIntoPackFile);
                     activity.RelatedEvent(EventLevel.Informational, this.Area, metadata, Keywords.Telemetry);

```

Comment:
Minor (and not strictly related to these changes), if you make the `eventName` parameter of `RelatedEvent` more unique it might be easier to filter.

*Example:*

Instead of:
`this.Area`

Something like:
`$"{this.Area}_{nameof(this.PerformMaintenance)}"`